### PR TITLE
Update certificate-signing-requests.md, include cert export

### PR DESCRIPTION
**Pull request by mpatters72**
Updating the instructions to include missing step of getting issued cert decoded, and ready locally before proceeding with kubeconfig setup.

### CODE REVIEW

**Comment by sftim** [comment]
Welcome!

also
/sig auth

**Review by sftim — COMMENTED** [review]
Here's some feedback. I hope it's useful.

**Comment by mpatters72** [comment]
@xiangpengzhao anything else needed to review?

**Comment by sftim** [comment]
@kubernetes/sig-auth-pr-reviews is this change good to go?

File: content/en/docs/reference/access-authn-authz/certificate-signing-requests.md [hunk 1]
```diff
@@ -304,7 +304,13 @@ kubectl create rolebinding developer-binding-john --role=developer --user=john
 
 The last step is to add this user into the KubeConfig. We assume the key and crt files are located here "/home/vagrant/work/".
 
-First, we need to add new credentials
+First, export the issued certificate from the CertificateSigningRequest:
```
Comment (liggitt) [inline content/en/docs/reference/access-authn-authz/certificate-signing-requests.md:307]: would this make sense to move to the `Get the Certificate` step above?

**Comment by zparnold** [comment]
@mpatters72 what did you think about @liggitt 's feedback?

File: content/en/docs/reference/access-authn-authz/certificate-signing-requests.md (hunk 1 above)
Comment (mpatters72) [reply content/en/docs/reference/access-authn-authz/certificate-signing-requests.md]: makes sense

**Comment by mpatters72** [comment]
Apologies for using name `@` mention in a commit.
Not sure if it would be necessary to start a fresh PR to move forward.
Please let me know.

**Comment by kbhawkey** [comment]
Hi @mpatters72 .
To move forward, would you rebase your changes into a single commit and amend the commit message (if needed)?
Thanks!

**Comment by mpatters72** [comment]
I didn't have much luck rebasing, so submitting as a new PR: https://github.com/kubernetes/website/pull/25735

**Pull request closed without merging**